src/client/entity-editor/common/linked-entity.tsx

```diff
@@ -77,13 +79,15 @@ class LinkedEntity extends React.Component<any, any> {
 	}
 
 	render() {
-		const option = this.getSafeOptionValue(this.props.option);
+		const option = this.getSafeOptionValue(this.props.data);
+		const styles = this.props.getStyles('option', this.props);
```

Comment:
This is going to throw an error if getStyles is not defined.
For example when used in `src/client/entity-editor/edition-section/edition-section.tsx`.

Another case where we're tying a component to react-select implementation details when we shouldn't be.
I'd rather see a separate component for use with react-select.

Reply:
I also believe we might as well have separate components for react-select.